templates/_images/opensuse-leap-16.yaml

```diff
@@ -0,0 +1,17 @@
+images:
+# Hint: run `limactl prune` to invalidate the Current cache
+
+- location: https://download.opensuse.org/distribution/leap/16.0/appliances/Leap-16.0-Minimal-VM.x86_64-Cloud.qcow2
+  arch: x86_64
+
+- location: https://download.opensuse.org/distribution/leap/16.0/appliances/Leap-16.0-Minimal-VM.aarch64-Cloud.qcow2
+  arch: aarch64
+
+- location: https://download.opensuse.org/distribution/leap/16.0/appliances/Leap-16.0-Minimal-VM.s390x-s390x-Cloud.qcow2
+  arch: s390x
```

Comment:
(This s390x image does not seem to execute cloud-init)

Reply:
I've asked questions, but I have not received responses so far.